Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM mesosphere/dcos-system-test-driver:latest
+
+# Specify the component versions to use
+ENV CYPRESS_VERSION="0.19.1" \
+    NODE_VERSION="4.4.7" \
+    NPM_VERSION="3.9"
+
+# Expose the 4200 port
+EXPOSE 4200
+
+# Copy required files in order to be able to do npm install
+WORKDIR /dcos-ui
+COPY package.json npm-shrinkwrap.json scripts /dcos-ui/
+
+# Copy the entrypoint script that takes care of executing run-time init tasks,
+# such as creating the scaffold in the user's repository
+COPY scripts/docker-entrypoint /usr/local/bin/dcos-ui-docker-entrypoint
+
+# Install required components & prepare environment
+RUN set -x \
```

Comment:
I would prefer to avoid creating an extra layer. I don't think we are saving much by doing so...